fapi-client/src/main/scala/com/gu/facia/api/utils/ResolvedMetaData.scala

```diff
@@ -0,0 +1,99 @@
+package com.gu.facia.api.utils
+
+import com.gu.contentapi.client.model.Content
+import com.gu.facia.api.models._
+import com.gu.facia.client.models.{TrailMetaData, MetaDataCommonFields}
+
+
+object ResolvedMetaData {
+  val Cartoon = "type/cartoon"
+  val Video = "type/video"
+  val Comment = "tone/comment"
+
+  def isCartoonForContent(content: Content): Boolean =
+    content.tags.exists(_.id == Cartoon)
+
+  def isCommentForContent(content: Content): Boolean =
+    content.tags.exists(_.id == Comment)
+
+  def isVideoForContent(content: Content): Boolean =
+    content.tags.exists(_.id == Video)
+
+  val Default = ResolvedMetaData(
+    isBreaking = false,
+    isBoosted = false,
+    imageHide = false,
+    imageReplace = false,
+    showKickerSection = false,
+    showKickerCustom = false,
+    showBoostedHeadline = false,
+    showMainVideo = false,
+    showKickerTag = false,
+    showByline = false,
+    imageCutoutReplace = false,
+    showQuotedHeadline = false)
+
+  private[utils] def fromTrailMetaData(trailMeta: MetaDataCommonFields): ResolvedMetaData =
+    ResolvedMetaData(
+      isBreaking = trailMeta.isBreaking.exists(identity),
+      isBoosted = trailMeta.isBoosted.exists(identity),
+      imageHide = trailMeta.imageHide.exists(identity),
+      imageReplace = trailMeta.imageReplace.exists(identity),
+      showKickerSection = trailMeta.showKickerSection.exists(identity),
+      showKickerCustom = trailMeta.showKickerCustom.exists(identity),
+      showBoostedHeadline = trailMeta.showBoostedHeadline.exists(identity),
+      showMainVideo = trailMeta.showMainVideo.exists(identity),
+      showKickerTag = trailMeta.showKickerTag.exists(identity),
+      showByline = trailMeta.showByline.exists(identity),
+      imageCutoutReplace = trailMeta.imageCutoutReplace.exists(identity),
+      showQuotedHeadline = trailMeta.showQuotedHeadline.exists(identity))
+
+  private[utils] def fromContent(content: Content): ResolvedMetaData = {
+    val mutatingMetaDataDefaults = Default
```

Comment:
Why not inline this?

e.g.

    if (isCartoonForContent(content))
      Default.copy(showByline = true)